spirv_parser.cpp

```diff
@@ -975,9 +978,16 @@ void Parser::parse(const Instruction &instruction)
 		current_block->condition = ops[0];
 		current_block->default_block = ops[1];
 
+		uint32_t remaining_ops = length - 2;
```

Comment:
Don't parse 32-bit case labels if it does not align to remaining_ops % 2 == 0

Reply:
I was thinking on having the `remaining_ops % 2` be the fallback case always, and when doing the branch fix see if it's necessary to override that case. Technically we "don't" have the option to choose since when one case is valid the other one is immediately invalid, so that's why I thought this way. Let me know what you think.